CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 * Add `AllowedGroups` configuration option to `RSpec/NestedGroups`. ([@ydah][])
 * Deprecate `IgnoredPatterns` option in favor of the `AllowedPatterns` options. ([@ydah][])
 * Add `AllowedPatterns` configuration option to `RSpec/ContextWording`. ([@ydah][])
+* Fix a false negative for `RSpec/SubjectStub` when the subject is declared with the `subject!` method and called by name. ([@eikes][])
```

Comment:
You have to add yourself to a list of contributors at the end of this file.

Reply:
Done!